src/modes/base.mode.ts

```diff
@@ -63,18 +69,12 @@ export abstract class TerraDrawBaseDrawMode<T extends CustomStyling> {
 	protected setCursor!: TerraDrawModeRegisterConfig["setCursor"];
 	protected registerBehaviors(behaviorConfig: BehaviorConfig): void {}
 
-	constructor(options?: {
-		styles?: Partial<T>;
-		pointerDistance?: number;
-		coordinatePrecision?: number;
-	}) {
+	constructor(options?: BaseModeOptions<T>) {
 		this._state = "unregistered";
 		this._styles =
 			options && options.styles ? { ...options.styles } : ({} as Partial<T>);
-
 		this.pointerDistance = (options && options.pointerDistance) || 40;
-
-		this.coordinatePrecision = (options && options.coordinatePrecision) || 9;
+		this.coordinatePrecision = 9;
```

Comment:
I think was mean to be left as is? I will merge and make another small PR to remedy this to help move it along :)